.github/workflows/linear-close-on-merge.yml

```
@@ -0,0 +1,175 @@
+
+jobs:
+  close-linear-ticket:
+    name: Close Linear Ticket
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Extract ticket ID from branch name
+        id: extract
+        shell: bash
+        run: |
```

actionlint (1.7.11)

[error] "github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)